platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/ConfigurationAdminImpl.java

```diff
@@ -178,10 +178,12 @@ private Bundle getBoundBundle(Configuration config) {
       // Get ManagedService instances
       serviceList = getServices(ManagedService.class.getName(), serviceFilter, true);
 
+      Map<String, ObjectClassDefinition> configPidToOcdMap =
```

Comment:
❗️ The `services` attribute at the top should either be defined as a `ConcurrentHashMap` or be synchronized as it is bound to be accessed by multiple threads and we are using it to lazily initialize the service tracker later.

Reply:
Since this is not related to changes of the PR, I will put up a separate PR to address this so that this back-port can remain clean.